dirs/dirs.go

```diff
@@ -151,6 +152,7 @@ func SetRootDir(rootdir string) {
 	SnapSocket = filepath.Join(rootdir, "/run/snapd-snap.socket")
 
 	SnapAssertsDBDir = filepath.Join(rootdir, snappyDir, "assertions")
+	SnapContextDir = filepath.Join(rootdir, snappyDir, "context")
```

Comment:
Is this in `/var/lib/snapd`? If it is, it's already covered: https://github.com/snapcore/snapd/blob/master/data/selinux/snappy.fc#L40